include/swift/AST/DiagnosticsSema.def

```diff
@@ -7871,12 +7871,11 @@ ERROR(lifetime_dependence_only_on_function_method_init_result, none,
 ERROR(lifetime_dependence_invalid_return_type, none,
       "lifetime dependence can only be specified on ~Escapable results", ())
  ERROR(lifetime_dependence_cannot_infer_ambiguous_candidate, none,
-      "cannot infer lifetime dependence, multiple ~Escapable or ~Copyable "
-      "parameters with ownership modifiers, specify explicit lifetime "
-       "dependence", ())
-ERROR(lifetime_dependence_cannot_infer_no_candidates, none,
-       "cannot infer lifetime dependence, no parameters with ownership "
-       "modifiers present", ())
+      "cannot infer lifetime dependence, multiple parameters qualifiy as a candidate", ())
+ ERROR(lifetime_dependence_cannot_infer_no_candidates, none,
+       "cannot infer lifetime dependence, no parameters found that are "
+       "~Escapable or Escapable with a borrowing ownership",
```

Comment:
This is still ambiguous. Maybe

`no parameters found that are either ~Escapable or have borrowing ownership.`

Reply:
I'll do this as a followup